src/lxc/confile.c

```diff
@@ -929,6 +931,24 @@ static int set_config_init_gid(const char *key, const char *value,
 	return 0;
 }
 
+static int set_config_oom_score_adj(const char *key, const char *value,
+			       struct lxc_conf *lxc_conf, void *data)
+{
+	int oom_score_adj;
+
+	if (lxc_config_value_empty(value)) {
+		lxc_conf->oom_score_adj = 0;
```

Comment:
You should just call your `clr_config_oom_score_adj()` function here and return:

```
if (lxc_config_value_empty(value))
        return clr_config_oom_score_adj()`
```